### DISCUSSION TITLE
Support Customer Relations module

### DISCUSSION THREAD

**Discussion by GregAujay** [python-gitlab/python-gitlab#2416] [Ideas]
Is it possible to support the customer relations module, with Contact and Organization, their related issues, etc.

**Comment by nejch** [comment]
@GregAujay I believe the CRM can only be edited via the GraphQL API, not via the REST API that this client currently supports. https://docs.gitlab.com/ee/user/crm/.

So you might want to wait for https://github.com/python-gitlab/python-gitlab/issues/1357, or use an existing graphql client such as https://github.com/graphql-python/gql.

But if you find REST API endpoints, please let me know here.
